test/all/run_all.py

```diff
@@ -137,7 +137,7 @@ def run(self):
                 lbuild_command += ["avr.xml"]
                 shutil.copyfile("avr.cpp", os.path.join(tempdir, "main.cpp"))
             else:
-                lbuild_command += ["cortex-m.xml", "-D:::main_stack_size=512"]
+                lbuild_command += ["cortex-m.xml", "-D:::main_stack_size=512", "-D:::main_fiber_stack_size=512"]
```

Comment:
It was kinda intended to reuse the main_stack_size, since that is now ignored (since there is no more non-fiber main()).

Reply:
I don't think it's ignored. Isn't there still a main stack where the ISRs execute and all code before fibers run?

Reply:
Well, at least on Cortex-M, AVRs don't have that option…

Reply:
Yes, only thought about cortex-m. There is hosted as well and potential other platforms we might support in the future, so this has to go to some platform specific setting. It is still useful to be able to configure both stack sizes on cortex-m. What do you suggest?

Reply:
I'm uncertain about this feature, since we cannot make AVR and hosted main a fiber, since we don't control how it's called. It would be inconsistent and confusing if the main function is a fiber only on Cortex-M. The only issue is when `modm::yield()` is called before the scheduler is initialized, but perhaps we can just spin in that case or modm_assert on it. I would remove the main fiber and make everything explicit, even if it will require refactoring some examples in the future.

Reply:
I'm unsure as well and would opt for consistency and removing the main fiber. That would also have the benefit of improved backwards compatibility for existing code with tiny 2-4kB RAM devices where splitting the stack to main + main fiber stack could be problematic.

docs generation is still broken: `Error generating documentation for device at90can64-16mu: 'NoneType' object is not subscriptable`

Reply:
This happens when lbuild has an error, usually when module dependencies are not satifed. I need to improve error reporting here…